api/src/test/java/org/pmiops/workbench/profile/ProfileServiceTest.java

```diff
@@ -29,14 +40,18 @@
 @DataJpaTest
 public class ProfileServiceTest {
 
+  @MockBean private InstitutionDao mockInstitutionDao;
+  @MockBean private InstitutionService mockInstitutionService;
   @MockBean private UserTermsOfServiceDao mockUserTermsOfServiceDao;
 
+  @MockBean
+  private VerifiedInstitutionalAffiliationMapper mockVerifiedInstitutionalAffiliationMapper;
+
   @Autowired ProfileService profileService;
   @Autowired UserDao userDao;
-  @Autowired VerifiedInstitutionalAffiliationMapper verifiedInstitutionalAffiliationMapper;
 
   @TestConfiguration
-  @MockBean({FreeTierBillingService.class})
+  @MockBean({FreeTierBillingService.class, InstitutionService.class, UserService.class})
```

Comment:
Why is InstitutionService included as a MockBean here and also above? I think you can remove one or the other, no?